src/Components/Slider.js

```diff
@@ -112,6 +118,7 @@ class CustomizedSlider extends React.Component {
         <Nouislider
           start={value}
           key={sliderKey}
+          connect
```

Comment:
@lambert3 , what's this used for if I may ask?

Reply:
It's for the temporal variation data. Its meant to replace the date picker. The earlier date picker wasn't depicting the quarters as stored in the database